deploy/lib/secret-traitor-stack.ts

```diff
@@ -87,31 +88,69 @@ export class SecretTraitorStack extends cdk.Stack {
         })
 
         ///////////// ACTUAL SECRET TRAITOR RESOURCES /////////////
-        const table = new dynamodb.Table(this, 'Games', {
-            tableName: 'Games',
-            partitionKey: { name: 'PK', type: dynamodb.AttributeType.STRING },
-            sortKey: { name: 'SK', type: dynamodb.AttributeType.STRING },
-        })
-
         // API Gateway REST API with the GraphQL API
-        const graphQLLambda = new lambda.Function(this, 'GraphQLHandler', {
-            runtime: lambda.Runtime.NODEJS_12_X,
+        const HttpHandler = new lambda.Function(this, 'HttpHandler', {
+            code: lambda.Code.fromAsset('../backend/dist'),
+            environment: { NODE_ENV: 'production' },
             handler: 'lambda.httpHandler',
+            memorySize: 512,
```

Comment:
As stated below, in my opinion there's no good reason to ever use a value less than 3008 MB for memorySize. I have often had Lambdas that contain whole Express apps throw _out of memory_ exceptions before increasing their memorySize to 3008.